src/components/Modals/DeleteSemester.vue

```diff
@@ -1,45 +1,30 @@
 <template>
-  <div class="deleteSemesterModal">
-    <div class="deleteSemesterModal-content">
-      <div class="deleteSemesterModal-top">
-        <span class="deleteSemesterModal-title">{{ title }}</span>
-        <button @click="closeCurrentModal">
-          <img
-            class="deleteSemesterModal-exit"
-            src="@/assets/images/x.png"
-            alt="x to close delete semester modal"
-          />
-        </button>
-      </div>
-      <div class="deleteSemesterModal-body">
-        <div class="deleteSemesterModal-body-text">{{ text }}</div>
-      </div>
-      <div class="deleteSemesterModal-buttonWrapper">
-        <button class="deleteSemesterModal-button" @click="closeCurrentModal">
-          {{ cancel }}
-        </button>
-        <button
-          class="deleteSemesterModal-button deleteSemesterModal-button--delete"
-          @click="deleteSemester"
-        >
-          <div class="deleteSemesterModal-button-left">
-            <img
-              class="deleteSemesterModal-button-left-icon"
-              src="@/assets/images/trash-white.svg"
-              alt="delete semester trashcan icon"
-            />
-            <span class="deleteSemesterModal-button-left-text">Delete</span>
-          </div>
-        </button>
-      </div>
+  <flexible-modal
+    title="Delete Semester"
+    class="deleteSemesterModal"
+    content-class="content-delete"
+    left-button-text="CANCEL"
+    right-button-text="DELETE"
```

Comment:
Can you do "Cancel" and "Delete"? It shouldn't be all caps

Reply:
Yep! It looks like we also use the capitalized text on the add course modal, should I change it there as well?

Reply:
Oh, and also on the see all buttons

Reply:
Woah good catch! Yeah I think changing it for the add course modal and see all buttons to be consistent makes sense. @einc do you agree?